install/prereqs/debian.rst

```diff
@@ -25,8 +25,26 @@ If you get an error run:
 
    apt-get update --fix-missing
 
-and try again.
+and run again the apt-get install above.
 
 .. note::
 
-   **New since 11.0**: The minimum gcc version required to compile the LSST Science Pipelines is **gcc 4.8.**
+   **New since 15.0**: The gcc compiler to be used shall support **C++ 14**. The gcc version used in Ubuntu 16 to compile the LSST Science Pipelines is **gcc 5.4.**.
+
+Additional installation steps for Ubuntu 14
+^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
+
+In the case you are deploying the lsst_distrib in an Ubuntu 14 machine, the gcc compiler installed will be the **4.8**. In order to get gcc version **5.4** following steps have to be completed in addition at the above ones.
```

Comment:
“If you are deploying lsst_distrib on an Ubuntu 14 system, the default gcc installed is version **4.8**”?

Also: what's Ubuntu 14? There was no release with that version. I think we mean Ubuntu 14.04 (and possibly the same instructions might apply to Ubuntu 14.10, but we don't support that).

Reply:
Our tests have been done on 14.04 version. I will correct the text. In any case, from my understanding we are not supporting Ubuntu. Frossie wants to anticipate problems from potential Ubuntu users. For this reason we are mentioning this here, as it has been done in the past releases.

Reply:
You're right, of course; “support” is an ill-defined word in any case.

I think it's pretty unlikely that anybody has actually tried this with 14.10, which is really the key point.